_includes/footer.html

```diff
@@ -18,6 +18,10 @@
                    target="_blank"><img
                         src="{{ '/assets/img/ZenMEM_dunkel.png' | relative_url }}"
                         alt="ZenMEM Logo"/></a>
+                <a class="m-1" href="https://www.text-plus.org/"
```

Comment:
I think we decided to *not* add the logo to the footer?

Reply:
Yes, that's right. I haven't had time to fix that, yet. I'll do it right now!